instance/models/openedx_appserver.py

```diff
@@ -176,6 +176,16 @@ class Meta:
             "behind a firewall. (In the django admin, separate group names with a comma.)"
         )
     )
+    additional_monitoring_emails = ArrayField(
```

Comment:
At some point we should consolidate how we represent list of strings in our models.  It's a bit all over the place at the moment.  For the list LMS users, we use a many-to-many relation (which is probably fine), for the github admin users we use a `JSONField`, and for this field we use an `ArrayField`.